app/templates/header.html

```diff
@@ -24,22 +24,35 @@
 		
 		  <div class="collapse navbar-collapse" id="navbarSupportedContent">
 		    <ul class="navbar-nav mr-auto">
-		      <li class="nav-item active">
-		        <a class="nav-link" href="{{url_for('flixlist', menu_item='Movies')}}">FlixList <span class="sr-only">(current)</span></a>
-		      </li>
-		      <li class="nav-item active">
-		        <a class="nav-link" href="">FlixHistory <span class="sr-only">(current)</span></a>
-		      </li>
-		      <li class="nav-item active">
-		        <a class="nav-link" href="">RecsFlix <span class="sr-only">(current)</span></a>
-		      </li>
-			  <li class="nav-item active">
-		        <a class="nav-link" href="">FriendFlix <span class="sr-only">(current)</span></a>
-		      </li>
-			  <li class="nav-item active">
-		        <a class="nav-link" href="">Login <span class="sr-only">(current)</span></a>
-		      </li>
-		  </div>
+				{% if current_user.is_authenticated %}
+				<li class="nav-item active">
+					<a class="nav-link" href="{{url_for('flixlist', menu_item='Movies')}}">FlixList <span class="sr-only">(current)</span></a>
+		      	</li>
+		      	<li class="nav-item active">
+		        	<a class="nav-link" href="">FlixHistory <span class="sr-only">(current)</span></a>
+		      	</li>
+		      	<li class="nav-item active">
+		        	<a class="nav-link" href="">RecsFlix <span class="sr-only">(current)</span></a>
```

Comment:
I thought the names of some menus had been changed

Reply:
Based on #17, only the file names were changed but not the front-end aspect. Will commit this now